src/mig/modules/pkgprint/paramscreator.go

```diff
@@ -0,0 +1,75 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Contributor: Aaron Meihm ameihm@mozilla.com [:alm]
+package pkgprint
+
+import (
+	"flag"
+	"fmt"
+)
+
+func printHelp(isCmd bool) {
+	dash := ""
+	if isCmd {
+		dash = "-"
+	}
+	fmt.Printf(`Query parameters
+----------------
+%stemplate <name>   - Scan using template
```

Comment:
Yeah that is correct, currently they are hard coded in the agent. That was the intent with the templateMode boolean in the parameters; if this is false a template from the command line would be used instead of a hardcoded one.

We could permit specifying a template via the command line -- in doing that though it opens up the ability to return information from the devices that could be sensitive (password hashes, etc). I'm not sure of a good way to work around this right now, we could do something like blacklist certain files or directories, I'm not sure that would be ideal either.